src/window.rs

```diff
@@ -1115,6 +1115,43 @@ impl Window {
     self.window.set_progress_bar(_progress)
   }
 
+  /// Sets the badge count on the taskbar
+  ///
+  /// ## Implementation Details
+  /// - _desktop_filename is for linux ONLY!
+  /// 
+  /// ## Platform-specific
+  ///
+  /// - **Windows**: Windows allows to set an arbitrary .ico file. See set_overlay_icon
+  /// - **Linux / macOS**: Badge count is app-wide and not specific to this window. Only supported desktop environments with `libunity` (e.g. GNOME).
+  /// - **iOS / Android:** Unsupported.
+  #[inline]
+  pub fn set_badge_count(&self, _count: Option<i64>, _desktop_filename: Option<String>) {
```

Comment:
I wonder if we can support this API on Windows by creating a simple icon programmatically that is red circle, with the number inside it.

Reply:
maybe, but will that not increase the complexity of the crate ?

Reply:
if we don't want to support it here, then I suggest moving `set_badge_count` to `WindowExtUnix` and `WindowExtMacos` since it is only supported on these two platforms and require different parameters anyways.

Reply:
✅